test/byContract/IexecAccessors/IexecAccessorsABILegacy.test.ts

```diff
@@ -0,0 +1,187 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+import { AddressZero } from '@ethersproject/constants';
+import { loadFixture } from '@nomicfoundation/hardhat-network-helpers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+import { ethers, expect } from 'hardhat';
+import { loadHardhatFixtureDeployment } from '../../../scripts/hardhat-fixture-deployer';
+import {
+    IexecInterfaceNative,
+    IexecInterfaceNativeABILegacy,
+    IexecInterfaceNativeABILegacy__factory,
+    IexecInterfaceNative__factory,
+} from '../../../typechain';
+import { OrdersActors, OrdersAssets, OrdersPrices, buildOrders } from '../../../utils/createOrders';
+import {
+    ContributionStatusEnum,
+    TaskStatusEnum,
+    buildResultHashAndResultSeal,
+    buildUtf8ResultAndDigest,
+    getIexecAccounts,
+} from '../../../utils/poco-tools';
+import { IexecWrapper } from '../../utils/IexecWrapper';
+
+const standardDealTag = '0x0000000000000000000000000000000000000000000000000000000000000000';
+const volume = 1;
+const trust = 1;
+const categoryId = 1;
+const appPrice = 1000;
+const datasetPrice = 1_000_000;
+const workerpoolPrice = 1_000_000_000;
+const callbackAddress = ethers.Wallet.createRandom().address;
+const dealParams = 'params';
+const { resultDigest } = buildUtf8ResultAndDigest('result');
+const taskIndex = 0;
+
+let proxyAddress: string;
+let iexecPoco: IexecInterfaceNative;
+let iexecPocoABILegacy: IexecInterfaceNativeABILegacy;
+let iexecWrapper: IexecWrapper;
+let [appAddress, workerpoolAddress, datasetAddress]: string[] = [];
+let [
+    requester,
+    beneficiary,
+    appProvider,
+    datasetProvider,
+    scheduler,
+    anyone,
+    worker1,
+]: SignerWithAddress[] = [];
+let ordersActors: OrdersActors;
+let ordersAssets: OrdersAssets;
+let ordersPrices: OrdersPrices;
+let [dealId, taskId, resultHash, resultSeal]: string[] = [];
+
+describe('IexecAccessorsABILegacy', function () {
+    beforeEach('Deploy', async () => {
+        // Deploy all contracts
+        proxyAddress = await loadHardhatFixtureDeployment();
+        // Initialize test environment
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        const accounts = await getIexecAccounts();
```

Comment:
Why not use destructuring directly after the result? Like 
```suggestion
        const {appPrivider, datasetProdvider,...} = await getIexecAccounts();
```

Reply:
`accounts` used in `IexecWrapper(proxyAddress, accounts)`.